.travis.yml

```diff
@@ -1,62 +1,58 @@
 language: php
 
 php:
-  - 5.3
-  - 5.4
-  - 5.5
-  - 5.6
-  - 7.0
-  - 7.1
-  - nightly
-  - hhvm-3.12
+    - 5.6
+    - 7.0
+    - 7.1
+    - nightly
+    - hhvm-3.12
 
 matrix:
-  allow_failures:
```

Comment:
https://docs.travis-ci.com/user/customizing-the-build
All examples give us indent with two spaces not four